notebooks/Revised_Example_LLH_setup_fixed_dirs.ipynb

```diff
@@ -0,0 +1,1528 @@
+    "\n",
+    "The calculation of the log likelihood is done through the LLH_webins class object, **sig_llh_obj**, which utilizes the event data (**ev_data0**), the lower and upper bounds of the energy bins (**ebins0** and **ebins1**), and the detector data (**bl_dmask**). **sig_llh_obj** makes use of the compound model by invoking the .set_model() function with **comp_mod** as an argument. When initializing **sig_llh_obj**, the argument has_err is set to True in order to utilize the error values set up in the compound model."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 96,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "sig_llh_obj = LLH_webins(ev_data0, ebins0, ebins1, bl_dmask, has_err=True)\n",
+    "sig_llh_obj.set_model(comp_mod)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The log likelihood minimization is handled by the NLLH_ScipyMinimize_Wjacob object, **sig_miner**, which is a wrapper of the scipy minimizer object. Using the .set_llh() function, it can act on **sig_llh_obj** to recalculate the log likelihood through variations of the parameters. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 97,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "sig_miner = NLLH_ScipyMinimize_Wjacob('')\n",
+    "sig_miner.set_llh(sig_llh_obj)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "In order to cut down on the computation time, only one parameter vary during this minimization process. The 'Signal_A' parameter will be varied while the other parameters are held fixed. The rest of the parameters will be held constant. The parameters to be kept constant are listed by **fixed_pnames** with corresponding values **fixed_vals**. A transformation on each of the parameter variables can be supplied in a list **trans**. In this scenario, all of the fixed parameteres are supplied without any transformations. \n",
+    "\n",
+    "The minimizer object saves these fixed parameters and transformations through the functions .set_trans() and .set_fixed_params(). Since 'Signal_A' is to be varied, .set_fixed_params() is used with the option fixed=False to inform the minimizer that this parameter will be varied over.\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 98,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "fixed_pnames = list(pars_.keys())\n",
+    "fixed_vals = list(pars_.values())\n",
+    "trans = [None for i in range(len(fixed_pnames))]\n",
+    "sig_miner.set_trans(fixed_pnames, trans)\n",
+    "sig_miner.set_fixed_params(fixed_pnames, values=fixed_vals)\n",
+    "sig_miner.set_fixed_params(['Signal_A'], fixed=False)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The source model flux parameters are known *a priori* with the gamma and Epeak values set at 0.8 and 350.0, respectively."
```

Comment:
"...are known a priori" -- speciffy that this is for this example analsyis. 
typically we do not know these values a priori